test/test_cross_resonance_hamiltonian.py

```diff
@@ -0,0 +1,318 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Spectroscopy tests."""
+
+import numpy as np
+from ddt import ddt, data, unpack
+from qiskit import QuantumCircuit, circuit, pulse
+from qiskit.providers.models import PulseBackendConfiguration
+from qiskit.result import Result
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeBackend
+
+from qiskit_experiments.library.characterization import cr_hamiltonian, cr_hamiltonian_analysis
+from qiskit_experiments.test.utils import FakeJob
+
+
+class CrossResonanceHamiltonianBackend(FakeBackend):
```

Comment:
Would it be possible to use something like `FakeOpenPulse2Q` to reduce the amount of boiler-plate code?

Reply:
We can use `ConfigurableBackend` but it has a bug. We can replace once it's fixed.

Reply:
https://github.com/Qiskit/qiskit-terra/pull/7022